<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResampleViewControls</class>
 <widget class="QWidget" name="ResampleViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
	border-width: 1px; 
	border-style: solid; 
	border-color: grey;
	border-radius: 5px;
	margin-top: 1ex;
}

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 10px;
}

QScrollArea{
	border: 0;
	margin:0;
	padding:0;
}

[mandatoryField=true]{
	background-color: rgb(255, 150, 150);
}

[warningField=true]{
	background-color: rgb(255, 204, 0);
}

[errorField=true]{
	background-color: rgb(255, 150, 150);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>357</width>
        <height>616</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="selectedImage">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;no image selected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Sagittal</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Coronal</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Axial</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSpinBox" name="originDim1">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="originDim2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="originDim3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="mul2Button">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>×2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="copyButton">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::RightArrow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="div2Button">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>÷2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QSpinBox" name="resampleDim1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="resampleDim2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="resampleDim3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Interpolation method</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="interpolatorComboBox">
                <item>
                 <property name="text">
                  <string>Linear</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Nearest neighbor</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gaussian</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BSpline</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resampleButton">
             <property name="text">
              <string>Resample</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="inplaceCheckBox">
             <property name="text">
              <string>Overwrite current image</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
